doc/guides/build/kconfig/menuconfig.rst

```diff
@@ -6,7 +6,10 @@ Interactive Kconfig interfaces
 There are two interactive configuration interfaces available for exploring the
 available Kconfig options and making temporary changes: ``menuconfig`` and
 ``guiconfig``. ``menuconfig`` is a curses-based interface that runs in the
-terminal, while ``guiconfig`` is a graphical configuration interface.
+terminal, while ``guiconfig`` is a graphical configuration interface. There is
```

Comment:
I would put this in its own section, this is really not related to 'interactive Kconfig interfaces'.

Reply:
That's true. We already have [a page under Security for the hardening tool](https://docs.zephyrproject.org/latest/security/hardening-tool.html), though - do you think we should move that documentation under Kconfig and mention it under Security, or leave it where it is and mention it in one of the other Kconfig pages (not Interactive Kconfig interfaces)? @ceolin

Reply:
Yep, the interactive section is not the right place. Though I think it is good link the hardening section in Kconfig section. May add a link to that in:
https://docs.zephyrproject.org/latest/guides/build/index.html#configuration-system-kconfig ?

Reply:
Moved mention to main Kconfig page :+1: